superset/queries/saved_queries/api.py

```diff
@@ -105,9 +105,13 @@ class SavedQueryRestApi(BaseSupersetModelRestApi):
         "sql",
         "sql_tables",
         "rows",
+        "tags.id",
+        "tags.name",
+        "tags.type",
         "last_run_delta_humanized",
         "extra",
     ]
+    list_select_columns = list_columns + ["changed_by_fk", "changed_on"]
```

Comment:
This is unrelated to the tagging stuff, but it seems it was missing from here and it was crashing my queries with the tags fields in list_columns without it.